library.js

```diff
@@ -81,6 +81,7 @@ plugin.updateUser = function(data, callback) {
 plugin.onForceEnabled = function(users, callback) {
 	if (plugin.hasOwnProperty('settings') && plugin.settings.force === 'on') {
 		async.map(users, function(userObj, next) {
+			if (!userObj) return;
```

Comment:
You cannot simply call `return` here, unfortunately. Replace this with `setImmediate(next, null, userObj);` please